public/app/features/explore/RichHistory/RichHistoryStarredTab.tsx

```diff
@@ -8,9 +8,7 @@ import { RichHistoryQuery, ExploreId } from 'app/types/explore';
 // Utils
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to `app/core/utils/richHistory`.